utilities/src/main/kotlin/org/jetbrains/kotlin/cli/utilities/InteropCompiler.kt

```diff
@@ -27,22 +27,17 @@ fun invokeInterop(flavor: String, args: Array<String>): Array<String>? {
     val temporaryFilesDir = arguments.tempDir
 
     val buildDir = File("$outputFileName-build")
-    val generatedDir = File(buildDir, "kotlin")
-    val nativesDir = File(buildDir, "natives")
+    val generatedDir = File(arguments.generated)
+    val nativesDir = File(arguments.natives)
```

Comment:
Do I understand correctly that `generatedDir` and `nativesDir` aren't located under `buildDir` anymore when calling `cinterop`?

Reply:
They are under if you use default values. I changed default values, and now default value of these parameters are created like `nativelib-build/kotlin`. The only problem is that if reset output parameter you have to reset these two also.
Unfortuantly, that's impossible now to show dependency on `kotlinx.cli`

Reply:
But it seems better to me to use such approach than just redefine provided by user arguments

Reply:
> They are under if you use default values.

Is this true if you use non-default output (which is almost always the case)?

Reply:
You should reset them if you use not default output. There are predefined parameters from native interop plugin 
```
File getNativeLibsDir() {
        return new File(project.buildDir, "nativelibs/$target")
    }

    File getGeneratedSrcDir() {
        return new File(project.buildDir, "nativeInteropStubs/$name/kotlin")
    }
```

Reply:
Ok, I see problem in MPP gradle plugin. Will it be ok if I fix there by adding not default parameters?

Reply:
Or is it better, to change command line to allow to change values of generated and natives only inside buildDir?

Reply:
https://github.com/JetBrains/kotlin/commit/f5f8dfac708d0aae8d7720d2431b8f642044ca82

Reply:
What I mean is
```
$ cinterop -def test.def -o test.klib
```
creates **both** `nativelib-build` and `test.klib-build` with your patch. It wasn't acting like this before.

Reply:
Yes, now you have to add `-generated test.klib/kotlin -natives test.klib/natives`.I made such changes in MPP plugin. Is this critical?

Reply:
> Is this critical?

Yes. It makes CLI cumbersome for the default use-case.
BTW, `CommonInteropArguments.generated` is not needed for `metadata` mode.

Reply:
Ok, added new feature to kotlinx.cli, but I'll wait review from Ilya G. https://github.com/Kotlin/kotlinx.cli/pull/22/files